tools/acrntrace/README.rst

```diff
@@ -18,8 +18,11 @@ output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
 Options:
 
 -h			print this message
--t period		specify polling interval in milliseconds [1-999]
+-i period		specify polling interval in milliseconds [1-999]
+-t			max time to capture trace data (in second)
 -c			clear the buffered old data
+-r			minimal amount (in MB) of free space kept on the disk
```

Comment:
same here, mention the parameter:

    -r space          amount of free space (in MB) remaining on the disk
                      before acrntrace stops